app/models/dhs1171_pdf.rb

```diff
@@ -14,20 +14,25 @@ class Dhs1171Pdf
     "zip",
   ].freeze
 
-  SOURCE_PDF = "DHS_1171.pdf".freeze
+  PDF_DIRECTORY = "lib/pdfs".freeze
+  SOURCE_PDF = "#{PDF_DIRECTORY}/DHS_1171.pdf".freeze
+  COVERSHEET_PDF = "#{PDF_DIRECTORY}/michigan_snap_fax_cover_letter.pdf".freeze
 
-  def initialize(client_data)
+  def initialize(client_data:, output_filename: "test_output.pdf")
     @client_data = client_data
+    @working_filename = "tmp/working_#{output_filename}"
```

Comment:
There is always a better name! I was thinking that I want the final PDF with the PDF to have the same name that was passed into the constructor. That's why I created some kind of interim "working" file.

What about renaming to `completed_form` or similar?